CONTRIBUTING.md

```diff
@@ -134,14 +134,13 @@ You can check the last publication date here: https://s01.oss.sonatype.org/conte
 
 Just do the following:
 
-```
+```sh
 git clone git@github.com:quarkusio/quarkus.git
 cd quarkus
-export MAVEN_OPTS="-Xmx4g"
-./mvnw -Dquickly
+MAVEN_OPTS="-Xmx4g" ./mvnw -Dquickly
```

Comment:
We did use an `export` on purpose. This way you don't have to do it every time you build the tree.

But I won't fight for it if you think this is better in this context.

Reply:
But the export only survives the current terminal session, no? 
I usually find that when that is done, the next time someone tries to execute the command that follows the `export`, it will fail and people don't know why.